CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## 1.3.0
+
+Removed `active_model` version dependency of `<5.2.0`
+Fixed `cancancan` version dependecy of `<=2.1.4`
```

Comment:
you have never released a gem with those version limitations? Because I don't see the change removing this dependency.

Reply:
The change removing this dependency was part of another PR, but it was not released yet.

Reply:
You should've released a version with limitation the day you discovered the issue as people have been able to install the broken version combination.